src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -52,6 +52,20 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
     return E_FAIL;
   }
 
+ #if defined(ARM64) || defined(ARM)
+  //
+  // In ARM64 and ARM, complete ReJIT support is only available from .NET 5.0
+  //
+  ICorProfilerInfo12* info12;
+  HRESULT hrInfo12 = cor_profiler_info_unknown->QueryInterface(__uuidof(ICorProfilerInfo12), (void**)&info12);
+  if (SUCCEEDED(hrInfo12)) {
+    Info(".NET 5.0 runtime or greater was detected.");
+  } else {
+    Warn("DATADOG TRACER DIAGNOSTICS - Profiler disabled: .NET 5.0 runtime or greater is required on this architecture.");
```

Comment:
In the log file, is ARM or ARM64 printed out anywhere?

Reply:
Yes here: https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace.ClrProfiler.Native/class_factory.cpp#L64